<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>475</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>846</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color:#000000</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #2b2b2b;     
    color: #ff4c4c;              
    border: 2px solid #cc0000;     
    padding: 4px;
    font-size: 12px;
    font-weight: bold;
    border-radius: 6px;
}</string>
         </property>
         <property name="text">
          <string>Registro de Usuarios</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
    color: #ff4c4c;              
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 1.3px;
}</string>
         </property>
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNombre">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;                
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>Apellidos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineApellidos">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;                 
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEmail">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;                
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>DNI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDNI">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;              
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>Nombre de Usuario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUsuario">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;             
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>Contrase√±a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineContrasena">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #2b2b2b;      
    color: white;                   
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
    selection-background-color: #444444;
    selection-color: #ff4c4c;        
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #ff4c4c;               
    font-size: 10px;
    font-weight: bold;
    padding-bottom: 2px;
}</string>
         </property>
         <property name="text">
          <string>Tipo de Usuario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxTipoUsu">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: #2b2b2b;   
    color: white;                    
    border: 1.5px solid #777777;     
    padding: 2px;
    border-radius: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #777777;
    background-color: #3a3a3a;
}

QComboBox QAbstractItemView {
    background-color: #2b2b2b;
    color: white;
    selection-background-color: #444444;
    selection-color: #ff4c4c;     
    border: 1px solid #555555;
}</string>
         </property>
         <item>
          <property name="text">
           <string>Sencillo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Premium</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCrear">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 3px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
         </property>
         <property name="text">
          <string>Crear Usuario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancelar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 3px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>